### ISSUE TITLE
Update Lesson Not Working

### ISSUE THREAD

**Issue by minglho** [isle-project/isle-editor#1136]
I uploaded a revised lesson. When I accessed it online through the ISLE dashboard, the following message appeared.

![image](https://user-images.githubusercontent.com/13249661/87212931-28b59480-c2d6-11ea-8534-c998ff71329e.png)

I clicked on the `here` button to update, but nothing happened. The lesson shown is an old lesson. For the updated lesson to show up, I had to clear the cache in Chrome. I am assuming that, if the `here` button had worked, that's what it would have done. 

**Desktop (please complete the following information):**
 - OS: Windows 10 
 - Browser: Chrome Version 83.0.4103.116 (Official Build) (64-bit)
 - ISLE Version 0.57.1

**Comment by Planeshifter** [comment]
Yes, your assumption is correct that clicking the button should have triggered a hard refresh. I was able to to replicate the issue and have worked on a fix that solves this problem, which will be included in the next editor release. 
Background of this is that we recently added support for offline usage of ISLE lessons through the use of service workers for students who might not continuously have a good Internet connection. Lessons should be accessible even if offline as long as they were loaded before and can also be installed locally ("+" button to the right of address bar in Chrome). 

Leaving this issue open until new editor is out and we have confirmed that the problem is indeed gone. Thanks for reporting!
